src/main.rs

```diff
@@ -1197,15 +1223,58 @@ fn cmd_vet(out: &mut dyn Write, cfg: &Config) -> Result<(), VetError> {
     trace!("vetting...");
 
     let mut store = Store::acquire(cfg)?;
+    let network = Network::acquire(cfg);
+
     if !cfg.cli.locked {
```

Comment:
We may also want a flag to override this without specifying locked at some point.